### ISSUE TITLE
Ventoy cannot "Install Ubuntu alongside other partitions"

### ISSUE THREAD

**Issue by davidhedlund** [ventoy/Ventoy#2595]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.95

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

Yes. I have tried them, but the bug still exist.

### BIOS Mode

UEFI Mode

### Partition Style

GPT

### Disk Capacity

120GB

### Disk Manufacturer

Kingston

### Image file checksum (if applicable)

Yes.

### What happened?

The below steps are done on cheap small hard disk drives instead of USB devices, because they have faster write speed.

Steps to reproduce
* Install Ventoy on a small hard disk drive from now on called Disk A. I used a 32 GB SATA SDD, but they are hard to find. 120 GB SATA SSDs are still easy to get possible to get at this writing.
* Make a disk image of Disk A, call it ventoy32gb.img

For a larger disk, from now on called Disk B. I used a 120 GB SATA SSD.
* Format the disk
* Restore the ventoy32gb.img disk image to the disk. 88 GB disk space will be free.
* Mount the Ventoy partition that now is available from the disk.
* Shut down the computer. Detach all drives (NVMe, SATA, PCIe, USB, etc) except the Disk B.
* Start the computer.
* Reboot -- Ventoy will boot from Disk B.
* Select the Ubuntu image from the Ventoy boot list.
* When Ubuntu starts, select "Install Ubuntu"
* During the Ubuntu installation, select "Install Ubuntu alongside other partitions"
* Despite that 120GB is free, the installation fails.

![IMG_20231006_052432](https://github.com/ventoy/Ventoy/assets/6416073/a41f04ad-03d8-49e8-b1e5-6077c7a6ad03)
![IMG_20231006_052529](https://github.com/ventoy/Ventoy/assets/6416073/c021f0aa-505d-4b09-8eb1-6e7de81fd796)
![IMG_20231006_052655](https://github.com/ventoy/Ventoy/assets/6416073/8ae278bf-815f-4d23-a839-7f5c2c9401f5)

**Comment by steve6375** [comment]
Why not just install Ventoy to the Drive B? In the Ventoy2DIsk.exe utility you just reserve some space.
Do these instructions apply to both Windows and Linux?

**Comment by davidhedlund** [comment]
> Why not just install Ventoy to the Drive B? In the Ventoy2DIsk.exe utility you just reserve some space. Do these instructions apply to both Windows and Linux?

You are right. Actually, I learned about this earlier today: "Option -> Partition Configuration". Check "Preserve some space at the end of the disk". I updated the steps in the top post so it's much less hazzle now.

>  Do these instructions apply to both Windows and Linux?

I have only tried it in GNU/Linux.

**Comment by davidhedlund** [comment]
* https://github.com/ventoy/Ventoy/issues/2605
